src/wasm_udf/data_types.rs

```diff
@@ -62,28 +62,22 @@ pub enum CreateFunctionDataType {
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, EnumString, Display, Clone)]
 #[serde(rename_all = "camelCase")]
+#[derive(Default)]
 pub enum CreateFunctionVolatility {
     Immutable,
     Stable,
+    #[default]
     Volatile,
 }
-impl Default for CreateFunctionVolatility {
-    fn default() -> Self {
-        CreateFunctionVolatility::Volatile
-    }
-}
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, EnumString, Display, Clone)]
 #[serde(rename_all = "camelCase")]
+#[derive(Default)]
 pub enum CreateFunctionLanguage {
+    #[default]
```

Comment:
I realize this is just `clippy` rewriting the existing code, but can we make `WasmMessagePack` the default?